containerTools.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# CONTAINER tools (e.g. docker, sarus, singularity)
+
+##################################################
+# functions
+##################################################
+function get_container {
```

Comment:
Will we need some kind of caching mechanism mid-term?

Reply:
Not sure what you are asking? I'm also not sure if this function makes sense to do -- it's not super general because you still would need to pass a different REMOTE_LOCATION based on whether it was docker or sarus (e.g. a GCR image vs )...